core/src/main/scala/org/apache/spark/deploy/history/FsHistoryProvider.scala

```diff
@@ -128,6 +128,10 @@ private[history] class FsHistoryProvider(conf: SparkConf, clock: Clock)
   private val storePath = conf.get(LOCAL_STORE_DIR).map(new File(_))
   private val fastInProgressParsing = conf.get(FAST_IN_PROGRESS_PARSING)
 
+  private val hybridStoreEnabled = conf.get(History.HYBRID_STORE_ENABLED)
+
+  private val memoryManager = new HistoryServerMemoryManager(conf)
```

Comment:
memory manager only appears to be used with hybrid store, so we shouldn't create and initialize it if not using the hybrid store

Reply:
Addressed.